### ISSUE TITLE
ios_interfaces does not add 'no shutdown' command when enabled: true

### ISSUE THREAD

**Issue by eleksis** [ansible-collections/cisco.ios#181]
##### SUMMARY

Module sometimes (not in every run) "forgets" to add `no shutdown` to newly created interface.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

ios_interfaces.py

##### ANSIBLE VERSION

```paste below
ansible 2.9.13
python version 3.7.3
cisco.ios 1.1.0
```

##### CONFIGURATION

```paste below
default
```

##### OS / ENVIRONMENT

Tested on Cisco IOS Software, IOS-XE Software, Catalyst 4500 L3 Switch Software (cat4500e-UNIVERSALK9-M), Version 03.10.01.E RELEASE SOFTWARE (fc4)

##### STEPS TO REPRODUCE

Configure/create new/non-existing Vlan interface with `enabled: true` (look below for detailed module args).

##### EXPECTED RESULTS

Module should add `no shutdown` command as this interface did not exist before (Cisco adds shutdown to new interface) and enabled:true is apssed in module args.
```
"commands": [
  "interface Vlan7",
  "description -- braavos_testmv --",
  "no shutdown"
]
```

##### ACTUAL RESULTS

```paste below
          {
            "hosts": {
              "4500x-switch": {
                "_ansible_no_log": false,
                "action": "cisco.ios.ios_interfaces",
                "after": [
                  {
                    "description": "Loopback0",
                    "enabled": true,
                    "name": "loopback0"
                  },
                  {
                    "duplex": "auto",
                    "enabled": true,
                    "name": "FastEthernet1",
                    "speed": "auto"
                  },
                  {
                    "description": "-- braavos --",
                    "enabled": true,
                    "name": "TenGigabitEthernet1/1"
                  },
                  {
                    "description": "-- volantis --",
                    "enabled": true,
                    "name": "TenGigabitEthernet1/2"
                  },
                  {
                    "enabled": true,
                    "name": "TenGigabitEthernet1/3"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/4"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/5"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/6"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/7"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/8"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/9"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/10"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/11"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/12"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/13"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/14"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/15"
                  },
                  {
                    "enabled": true,
                    "name": "TenGigabitEthernet1/16"
                  },
                  {
                    "description": "management",
                    "enabled": true,
                    "name": "Vlan1"
                  },
                  {
                    "description": "-- braavos_admin --",
                    "enabled": true,
                    "name": "Vlan3"
                  },
                  {
                    "description": "-- braavos_dmz --",
                    "enabled": true,
                    "name": "Vlan4"
                  },
                  {
                    "description": "-- braavos_eduroam --",
                    "enabled": true,
                    "name": "Vlan5"
                  },
                  {
                    "description": "-- braavos_testmv --",
                    "enabled": false,
                    "name": "Vlan7"
                  }
                ],
                "before": [
                  {
                    "description": "Loopback0",
                    "enabled": true,
                    "name": "loopback0"
                  },
                  {
                    "duplex": "auto",
                    "enabled": true,
                    "name": "FastEthernet1",
                    "speed": "auto"
                  },
                  {
                    "description": "-- braavos --",
                    "enabled": true,
                    "name": "TenGigabitEthernet1/1"
                  },
                  {
                    "description": "-- volantis --",
                    "enabled": true,
                    "name": "TenGigabitEthernet1/2"
                  },
                  {
                    "enabled": true,
                    "name": "TenGigabitEthernet1/3"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/4"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/5"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/6"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/7"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/8"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/9"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/10"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/11"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/12"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/13"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/14"
                  },
                  {
                    "enabled": false,
                    "name": "TenGigabitEthernet1/15"
                  },
                  {
                    "enabled": true,
                    "name": "TenGigabitEthernet1/16"
                  },
                  {
                    "description": "management",
                    "enabled": true,
                    "name": "Vlan1"
                  },
                  {
                    "description": "-- braavos_admin --",
                    "enabled": true,
                    "name": "Vlan3"
                  },
                  {
                    "description": "-- braavos_dmz --",
                    "enabled": true,
                    "name": "Vlan4"
                  },
                  {
                    "description": "-- braavos_eduroam --",
                    "enabled": true,
                    "name": "Vlan5"
                  }
                ],
                "changed": true,
                "commands": [
                  "interface Vlan7",
                  "description -- braavos_testmv --"
                ],
                "invocation": {
                  "module_args": {
                    "config": [
                      {
                        "description": "management",
                        "duplex": null,
                        "enabled": true,
                        "mtu": null,
                        "name": "Vlan1",
                        "speed": null
                      },
                      {
                        "description": "-- braavos_admin --",
                        "duplex": null,
                        "enabled": true,
                        "mtu": null,
                        "name": "Vlan3",
                        "speed": null
                      },
                      {
                        "description": "-- braavos_dmz --",
                        "duplex": null,
                        "enabled": true,
                        "mtu": null,
                        "name": "Vlan4",
                        "speed": null
                      },
                      {
                        "description": "-- braavos_eduroam --",
                        "duplex": null,
                        "enabled": true,
                        "mtu": null,
                        "name": "Vlan5",
                        "speed": null
                      },
                      {
                        "description": "-- braavos_testmv --",
                        "duplex": null,
                        "enabled": true,
                        "mtu": null,
                        "name": "Vlan7",
                        "speed": null
                      }
                    ],
                    "running_config": null,
                    "state": "replaced"
                  }
                }
              }
            },
```

**Comment by justjais** [comment]
@eleksis the issue is inactive and is waiting on the info we’re closing the issue, plz reopen a new issue if you’re still facing the reported issue
